### PR TITLE
HW03 Pull Request

### PR DESCRIPTION
**Pull request by eber-secco**
- Answered questions from HW03
- Create a code that shows a beautiful landscape of a tree and a house.

### CODE REVIEW

**Comment by eber-secco** [comment]
Hey sorry for trying to approve the pull request!

File: hw03_secolimae.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+# I also asked ChatGPT how to set the RBG color mode by saying "how I can use the RGB code style in the colors?"
+# I wasn't sure about the signal for docstrings so I searched it in this website: https://peps.python.org/pep-0257/
+#################################################################################
+
+import turtle
+
+wn = turtle.Screen() # Set the screen
+wn.screensize(600, 600)
+wn.colormode(255) # Define color mode to RGB
+wn.bgcolor((135, 206, 235)) # RGB color mode for blue
```
Comment (sheggen) [inline hw03_secolimae.py:21]: Move these four lines to the start of your `main()` function.

File: hw03_secolimae.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    create_trunk(t)
+    create_leaves(t)
+    create_walls(t)
+    create_roof(t)
+    create_door(t)
+    create_window(t)
+
+main()
+
+wn.exitonclick()
```
Comment (sheggen) [inline hw03_secolimae.py:163]: Move this up to after line 159, and indent it into `main()`

File: hw03_secolimae.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    t.goto(100, 100) # Put the turtle somewhere else
+
+
+def main():
+    wn = turtle.Screen()  # Set the screen
+    wn.screensize(600, 600)
+    wn.colormode(255)  # Define color mode to RGB
+    wn.bgcolor((135, 206, 235))  # RGB color mode for blue
+
+    """This function sets the turtle and create objects such as grass, trunk, leaves, walls, roofs, door, and window."""
```
Comment (sheggen) [inline hw03_secolimae.py:150]: The docstring should be after the `def main():` line.

**Comment by sheggen** [comment]
Code needs one last small fix. You also need to move one file:

1. Move **hw03_questions_secolimae.md** into the _Answers_ directory.

**Pull request merged**